src/content/protocols/aave-v3-etherfi.md

```diff
@@ -0,0 +1,518 @@
+---
+protocol: "Aave v3 EtherFi"
+website: "https://aave.com"
+x: "https://x.com/aave"
+github: ["https://github.com/aave-dao/aave-v3-origin"]
+defillama_slug: ["aave-v3"]
+chain: "Ethereum"
+stage: 0
+risks: ["L", "H", "H", "H", "L"]
+reasons: []
+author: ["sagaciousyves"]
+submission_date: "2025-05-16"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+On Ethereum Mainnet, 3 different instances exist _Core_, _Prime_ and _EtherFi_ which all are managed by the Aave DAO. **This report covers the Aave V3 _EtherFi_ instance.** To check out our main report on Aave V3 see the [_Core Instance_ report](/protocols/aave). This report focuses on the key differences between the _EtherFi_ and the _Core_ instance.
+
+Aave v3 is a lending protocol that allows users to lend and borrow different ERC20 assets. Users are able to create positions that consist of debt in different loan assets which is secured by different collateral assets. The lending market allows anyone to liquidate insolvent positions, based on an external price feed and specific collateral factors representing an asset's specific risk profile. Furthermore, instead of borrowing supplied assets, Aave V3 also issues its own stablecoin, `GHO`. Users can borrow and lend `GHO` like any other asset in the system.
```

Comment:
do you mean "in addition to lending supplied assets" and not "instead of borrowing supplied assets" ? If not I don't undertsand why

Reply:
yeah "in addition to lending supplied assets" is better, I mixed up perspectives of user and protocol